modules/cp4ba/outputs.tf

```diff
@@ -2,19 +2,19 @@ output "cp4ba_endpoint" {
   depends_on = [
     data.external.get_endpoints,
   ]
-  value = var.enable && length(data.external.get_endpoints) > 0 ? data.external.get_endpoints.0.result.endpoint : ""
+  value = var.enable_cp4ba && length(data.external.get_endpoints) > 0 ? data.external.get_endpoints.0.result.endpoint : ""
```

Comment:
Is `enable_cp4ba` in or out?  This `enable` flag isn't really needed for these cloud pak modules anymore, so ok to remove. It is implied that if this module is called from the sandbox scripts, it is wanted.  Unlike the helper modules like `portworx` or `db2`.